controlplane/test/scim.test.ts

```diff
@@ -0,0 +1,303 @@
+import { uid } from 'uid';
+import { afterAll, beforeAll, describe, expect, test } from 'vitest';
+import Keycloak from '../src/core/services/Keycloak.js';
+import { UserTestData, afterAllSetup, beforeAllSetup } from '../src/core/test-util.js';
+import { SetupKeycloak, SetupTest, removeKeycloakSetup } from './test-util.js';
+
+let dbname = '';
+let baseAddress = '';
+let realmName = '';
+let userTestData: UserTestData;
+let keycloakClient: Keycloak;
+let server: any;
+
+describe('Scim server', (ctx) => {
```

Comment:
Please document what SCIM version has been implemented.  Reference the spec.

Reply:
Done